src/MovieRequest.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import './ListStyle.css';
+import ReactDOM from 'react';
+import SaveToLS from './SaveObj';
+import DeleteFromLS from './DeleteObj';
+import ShowMovies from './ShowMovieList.js';
+import AddFavourite from './AddFavourites';
+
+export default function ListOfTop20(){
+    const [count, setCount] = useState(0);
+    const [isLoaded, setIsLoaded] = useState(false);
+    const [items, setItems] = useState([]);
+    const [error, setError] = useState(null);
+    const [favs, setFavs] = useState([]);
+
+    const TOKEN = '345afb1e-d1ea-4de9-80fd-9363d0200b32';
+
+    function AddFavMov(movie){

```

Comment:
Function name should start from small letter. Please check this function again. What will happen if I click on Fav button several time for the same movie? (Hint: it will be adeed several times too). You should prevent such cases and check if the movie are in the list already. If it is you should remove it from there, if not - add it.